service/routes.py

```diff
@@ -18,8 +18,27 @@
 @app.route("/")
 def index():
     """Root URL response"""
+
+    [{'id': 1, 'username': 'Alice'}, {'id': 2, 'username': 'Bob'}]
```

Comment:
We should remove this @raymondlu8547.  I assume you were using this to test jsonify?